spec/requests/targets/create_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'POST /targets', type: :request do
+
+  let!(:user)  { create(:user) }
+  let!(:topic) { create(:topic) }
+
+  let(:params) do
+    {
+      title:     Faker::Cannabis.buzzword,
+      topic_id:  topic.id,
+      latitude:  Faker::Number.decimal(l_digits: 2, r_digits: 2),
+      longitude: Faker::Number.decimal(l_digits: 2, r_digits: 2),
+      radius:    Faker::Number.between(from: 1, to: 5000)
+    }
+  end
+
+  subject { post targets_path, params: params, headers: auth_user_headers, as: :json }
+
+  it 'returns a successful response' do
+    subject
+    expect(response).to be_successful
+  end
+
+  it 'creates the target' do
```

Comment:
I would also test that the created target attribute values match the ones in the params, WDYT?